pkg/services/ngalert/notifier/file_store.go

```diff
@@ -7,8 +7,8 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/grafana/alerting/cluster/clusterpb"
```

Comment:
I think we should use a similar alias to what you see above. E.g. alertingCluster otherwise gofmt gets confused and it's easy to miss.